lib/resty/limit/req.lua

```diff
@@ -4,19 +4,19 @@
 local floor = math.floor
 local tonumber = tonumber
 
-
 local _M = { _VERSION = "0.01", OK = 1, BUSY = 2, FORBIDDEN = 3 }
 
-
-local redis_limit_req_script_sha
-local redis_limit_req_script = [==[
+local redis_limit_req_lookup_script_sha
+local redis_limit_req_lookup_script = [==[
```

Comment:
It will be loaded to redis as a string using [redis script load](https://redis.io/commands/script-load)